README.md

```diff
@@ -1,2 +1,8 @@
 # disqualified-officers-data-api
 Handles CRUD functions for disqualified officers.
+
+# Requirements
+
+- [Java 8](https://www.oracle.com/uk/java/technologies/javase/jdk11-archive-downloads.html)
```

Comment:
Shouldn't this be Java 11?

Reply:
Good point, I'll change the link text - the target url seems correct though